### PR TITLE
Reimplement upload inputs function

### PR DESCRIPTION
**Pull request by luispcunha**
- Uses urllib3 directly instead of the generated client to stream the file instead of loading it completely into memory before uploading
- Adds a tqdm progress bar

Stdout:
```

Preparing upload of the local input directory gromacs-input-example (2.15 GB).
Input archive size: 2.15 GB
Uploading input archive...
100%|█████████████████████████████████████████████████████████| 2.15G/2.15G [02:30<00:00, 14.3MB/s]
Local input directory successfully uploaded.
Task sf5d0n092y7fyp44aiu26zi4m submitted to the default queue.
```

### CODE REVIEW

**Comment by hpenedones** [comment]
This is awesome! Having the progress bar really helps as well.

**Review by sergiomsantos — APPROVED** [review]
Nice!!

File: inductiva/api/methods.py [hunk 1]
```diff
@@ -96,12 +100,45 @@ def upload_input(api_instance: TasksApi, task_id, original_params,
...
+
+            pool_manager.request(
+                "POST",
+                (f"{api_instance.api_client.configuration.host}"
+                 f"/tasks/{task_id}/input"),
+                body=wrapped_file,
+                headers={
+                    "X-API-Key": (api_instance.api_client.configuration.
+                                  api_key["APIKeyHeader"]),
+                },
```
Comment (AugustoPeres) [inline inductiva/api/methods.py:139]: If there is an error in the request. Is there any change that the clients api key can be exposed in the traceback?

**Review by AugustoPeres — APPROVED** [review]
LGTM. Left just a question regarding safety and api keys

**Pull request merged**